copy/copy.go

```diff
@@ -243,6 +275,15 @@ func copyBlob(dest types.ImageDestination, src types.ImageSource, srcInfo types.
 		return types.BlobInfo{}, fmt.Errorf("Error reading blob %s: %v", srcInfo.Digest, err)
 	}
 
+	if report {
+		bar := pb.New(int(srcInfo.Size)).SetUnits(pb.U_BYTES)
```

Comment:
Then the `report` bool could probably go away altogether, `copyBlob` would get an `io.Writer` or `nil`.